CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 1.4.0
```

Comment:
On a second thought, should you replace 1.4.0 with `devel` or `next-release`?

Reply:
Then we will update the version to `1.4.0-RCx`, then `1.4.0` once ready to release.